src/test/java/com/kasperin/inventory_management/services/ProcessedFoodServiceImplTest.java

```diff
@@ -29,55 +31,94 @@
     public static final FoodType FOODTYPE2 = FoodType.NONVEGAN;
 
     @InjectMocks
-    ProcessedFoodService processedFoodService;
+    ProcessedFoodServiceImpl processedFoodService;
```

Comment:
Mockito doesn't know what to do with an interface. `@InjectMocks` expects a concrete class